src/pages/Swap/LimitOrderBox/components/LimitOrderForm/LimitOrderForm.tsx

```diff
@@ -111,7 +114,7 @@ export function LimitOrderForm({ account, provider, chainId }: LimitOrderFormPro
     const cowQuote = await getQuote({
       chainId,
       signer,
-      order: { ...order, expiresAt: dayjs().add(expiresIn, expiresInUnit).unix() },
+      order: { ...order, expiresAt: dayjs().add(20, OrderExpiresInUnit.Minutes).unix() },
```

Comment:
I think 20 should not be hard coded and should be a variable (constant).

Reply:
I think it's fine since we want to get a market price for the pair, but your call @berteotti

Reply:
Will add it to a constant so we can easily manipulate all 20 minutes values.